PixelDefinitions/pixels/broken_site_report.json

```diff
@@ -0,0 +1,320 @@
+{
+    "epbf_android": {
+        "description": "User sent broken site report",
+        "owners": [],
```

Comment:
I think this will fail validation with the latest schema release (owners should not be empty). Maybe Kate can be the owner for Android breakage pixels? Let's ask her in Asana.